server/src/controllers/cars/updateCars.ts

```diff
@@ -0,0 +1,16 @@
+import { NextFunction, Request, Response } from 'express';
+import createError from 'http-errors';
+import { updateCarServes } from '../../services/cars';
+import updateCarSchema from '../../helpers/updateCarSchema';
+
+const updateCars = async (req: Request, res: Response, next:NextFunction) => {
+  const { body } = req;
+  const { id } = req.params;
+  updateCarSchema.validate(body).catch((err) => {
+    next(createError(404, err.errors));
+  });
```

Comment:
Why you do this, if there as an error it'll be thrown to `RouterWraper` where we handle validation error